sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitCoverageResultParser.java

```diff
@@ -48,32 +50,31 @@
 import org.sonar.plugins.php.phpunit.xml.PackageNode;
 import org.sonar.plugins.php.phpunit.xml.ProjectNode;
 
-/**
- * The Class PhpUnitCoverageResultParser.
- */
-public class PhpUnitCoverageResultParser implements BatchExtension, PhpUnitParser {
+@BatchSide
+@ExtensionPoint
+public class PhpUnitCoverageResultParser implements PhpUnitParser {
 
   // Used for debugging purposes to store measure by resource
-  private static final Map<Resource, Measure> MEASURES_BY_RESOURCE = new HashMap<>();
+  private static final Map<InputFile, Measure> MEASURES_BY_RESOURCE = new HashMap<>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Measure"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)